### ISSUE TITLE
React Native Crashes when an AnimatedLayout component is rendered without an animated view child

### ISSUE THREAD

**Issue by bailycase99** [software-mansion/react-native-reanimated#2296]
## Description

App crashing if you don't render an Animated View as a child of the AnimatedLayout

### Expected behavior

App to not crash, or throw a better error message

### Actual behavior & steps to reproduce

Render a <AnimatedLayout> without an <Animated.View> as a child.

## Snack or minimal code example

```jsx
<AnimatedLayout>
// <Animated.View />
   <Text>Some random text</Text>
</AnimatedLayout>
```

## Package versions

- React Native: 0.64.2
- React Native Reanimated: 2.3.0-alpha.1
- NodeJS: 15.14.0
- Xcode: 12.5.1
- Java & Gradle:

## Affected platforms

- [?] Android
- [x] iOS
- [ ] Web

**Comment by bailycase99** [comment]
It also occurs when you try to unmount the AnimatedLayout. We were able to resolve this by wrapping our whole app in the AnimatedLayout.

**Comment by AustinHunt** [comment]
A small correction, this only happens when you attempt to unmount an AnimatedLayout.

**Comment by AustinHunt** [comment]
![Screen Shot 2021-08-18 at 2 52 03 PM](https://user-images.githubusercontent.com/2763372/129955475-9e6331d6-a6c1-4231-a045-f4765b4d50de.png)

**Comment by jmysliv** [comment]
Hi! we are working currently on removing `AnimatedLayout` which should resolve this issue. It will be available in the next 2.3.0 alpha release.

**Comment by piaskowyk** [comment]
It should be fixed by: https://github.com/software-mansion/react-native-reanimated/pull/2302
